examples/analog_hamiltonian_simulation/09_Noisy_quantum_evolution_of_Bell_states.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.                yerr = [2 * np.sqrt(y)/shots for y in ys]
</pre><p>This formula for error bars is incorrect if y is a probablity. If <code>y = k / n</code>, with <code>k</code> being the number of one outcome and <code>n</code> is the total number of trials, then y_err (say 2 * the standard error) should be <code>y_err = 2 * sqrt(k) / n = 2 * sqrt(y / n)</code> and not <code>sqrt(y) / n</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>